cmd/config.go

```diff
@@ -381,35 +380,27 @@ func listConfig(stack backend.Stack, showSecrets bool) error {
 		decrypter = config.NewBlindingDecrypter()
 	}
 
-	fullKey := func(k config.Key) string {
-		return fmt.Sprintf("%s:%s", k.Namespace(), k.Name())
-	}
+	table := []cmdutil.TableRow{}
+	table = append(table, cmdutil.TableRow{Columns: []string{"KEY", "VALUE"}})
```

Comment:
Nit, any reason why not

```golang
table := []cmdutil.TableRow{
    {Columns: []string{"KEY", "VALUE"}},
}
```

? More idiomatic Go, methinks.

Reply:
i've squinted at this for like 5 minutes.  I can't see the difference between the two.  Can you clarify what change you'd like here? :)

Reply:
It just seems odd to initialize an empty array, only to append a constant element literal to it on the following line. Usually we just initialize the array with its initial contents.

Reply:
oh.  gotcha.  Sorry.  Sure.  Will make that change :)